docs/src/rules/no-this-before-super.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: no-this-before-super
 rule_type: problem
+handled_by_typescript:
+- 2376
```

Comment:
`2376` is _`A 'super' call must be the first statement in the constructor to refer to 'super' or 'this' when a derived class contains initialized properties, parameter properties, or private identifiers.`_. But there's also `17009`: _`'super' must be called before accessing 'this' in the constructor of a derived class.`_